cudf/bind/cudf_cpp.pxd

```diff
@@ -0,0 +1,1004 @@
+                                    gdf_column *keycol,
+                                    gdf_column *valcol)
+
+    # /* segmented sorting */
+    cdef gdf_segmented_radixsort_plan_type* gdf_segmented_radixsort_plan(size_t num_items, int descending,
+        unsigned begin_bit, unsigned end_bit)
+    cdef gdf_error gdf_segmented_radixsort_plan_setup(gdf_segmented_radixsort_plan_type *hdl,
+    size_t sizeof_key, size_t sizeof_val)
+    cdef gdf_error gdf_segmented_radixsort_plan_free(gdf_segmented_radixsort_plan_type *hdl)
+
+    # /*
+    # * The following function performs a sort on the key and value columns.
+    # */
+    cdef gdf_error gdf_segmented_radixsort_i8(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+    cdef gdf_error gdf_segmented_radixsort_i32(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+    cdef gdf_error gdf_segmented_radixsort_i64(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+    cdef gdf_error gdf_segmented_radixsort_f32(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+    cdef gdf_error gdf_segmented_radixsort_f64(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+    cdef gdf_error gdf_segmented_radixsort_generic(gdf_segmented_radixsort_plan_type *hdl,
+                                         gdf_column *keycol, gdf_column *valcol,
+                                         unsigned num_segments,
+                                         unsigned *d_begin_offsets,
+                                         unsigned *d_end_offsets)
+
+    # // joins
+
+
+    # /* --------------------------------------------------------------------------*/
```

Comment:
Why duplicate the block comments here if they are already in the C/C++ headers? Just makes for more duplication to maintain.

Reply:
I left them since during development it was useful to navigate the header file quickly, will delete them for the final PR.